Makefile

```diff
@@ -59,15 +62,28 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+gofmt-verify:
+	@out=`$(GO_FMT) -w -l -d $$(find . -name '*.go')`; \
+	if [ -n "$$out" ]; then \
+	    echo "$$out"; \
+	    exit 1; \
+	fi
+
+ci-lint:
```

Comment:
Add a make rule to install golangci-lint, like there is one for `kustomize` and other binaries.